_layouts/docs.html

```diff
@@ -17,9 +17,9 @@
         <div class="content .col-xs-12 .col-sm-8 .col-md-9">
             <h1>{{ page.title }}</h1>
             <div class="edit-container">
-                <a href="https://github.com/tableau/projectfrelard/edit/master/{{ page.path }}" class="edit-links"><span class="glyphicon glyphicon-pencil"></span> Edit this page</a>
+                <a href="https://github.com/tableau/extensions-api/edit/main/{{ page.path }}" class="edit-links"><span class="glyphicon glyphicon-pencil"></span> Edit this page</a>
```

Comment:
@bcantoni  Good catch!  Project Frelard was the code name and the name of the repo when we were first launching.

Reply:
Indeed - luckily GitHub does the right thing and handles with a redirect both the repo name and default branch name changes.